docs/guides/upgrading.rst

```diff
@@ -124,6 +124,13 @@ API to alter registration and login URL
  * ``registration_url, site`` hook can be used to alter the default registration URL
  * ``login_url, site`` hook can be used to alter the default login URL
 
+Support for fieldsets in forms
+------------------------------
+
+ * ``elgg_view_input()`` now supports ``#type``, ``#label``, ``#help`` and ``#class`` parameters to prevent name collision with input view ``$vars``
```

Comment:
Not to throw a wrench in this, but wouldn't the Drupal solution be to use `attributes` to feed attributes to the input element? That would be `input_vars` in our case. That would probably mean elgg_view_input doesn't change, and elgg_view_field would be the new way.